wp-content/themes/pub/wporg-learn-2024/src/sensei-lesson-actions-a11y/view.js

```diff
@@ -0,0 +1,14 @@
+function init() {
```

Comment:
This could be done with CSS — `display:none` hides content from everyone, including screen readers.

```css
.sensei-lesson-footer .screen-reader-text {
    display: none;
}
```

Reply:
Ah, yeah, I was initially thinking about completely removing it, so I went ahead and did it that way. Thanks!
2899198